stl/inc/atomic

```diff
@@ -840,6 +958,25 @@ struct _Atomic_storage<_Ty, 16> { // lock-free using 16-byte intrinsics
         return _Result != 0;
     }
 
+    void wait(_Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {
+        long _Spin_context = 0;
+        for (;;) {
+            _Ty _Observed = load(_Order);
+            if (_CSTD memcmp(&_Observed, _STD addressof(_Expected), sizeof(_Ty)) != 0)

```

Comment:
Maybe this should operate in the `_Int128` domain so that it's similar to the others? Kinda hard to show what would be better given that this `cmpxchg16b` area is deactivated at the moment...